.github/workflows/pullreq.yml

```
@@ -97,9 +97,18 @@ jobs:
         if: ${{ matrix.install_ninja }}
         uses: seanmiddleditch/gha-setup-ninja@master
 
-      - name: Build project
+      - name: Configure project (next)
+        if: github.ref == 'refs/heads/next'
+        run: |
+          cmake -S . -B ./build ${{ matrix.cmakeargs }} -DCMAKE_BUILD_TYPE=Debug -DCLAP_HELPERS_DOWNLOAD_DEPENDENCIES=TRUE -DCLAP_HELPERS_BUILD_TESTS=TRUE -DCLAP_HELPERS_DOWNLOAD_CLAP_BRANCH_IS_NEXT=TRUE
+
+      - name: Configure project (not next)
+        if: github.ref != 'refs/heads/next'
```

```suggestion
      - name: Configure project (main)
        if: github.ref == 'refs/heads/main'
```
Would this be better?

---

The reason I did it this way is if in the future we add a feature branch and want ci then it will work. But I thought about your way too. I’m not sure

---

feature branch make sense